server/request/request.go

```diff
@@ -324,26 +324,35 @@ func getMetadataFromToken(token string) (string, bool, string) {
 		log.Debug().Msg("Could not split the token into its parts")
 		return defaults.UnknownValue, false, ""
 	}
+
 	decodedToken, err := base64.RawStdEncoding.DecodeString(tokenParts[1])
 	if err != nil {
 		log.Debug().Err(err).Msg("Could not base64 decode token")
 		return defaults.UnknownValue, false, ""
 	}
-	namespace, err := jsonparser.GetString(decodedToken, "https://tigris/n", "code")
+
+	tigrisClaimObj, _, _, err := jsonparser.Get(decodedToken, "https://tigris")
```

Comment:
is it possible to add a unit test for this method by passing a token that is similar to what we expect?